dashboard/app/controllers/api/v1/pd/workshops_controller.rb

```diff
@@ -194,6 +194,42 @@ def summary
     render json: @workshop, serializer: Api::V1::Pd::WorkshopSummarySerializer
   end
 
+  # Users who could be re-assigned to be the organizer of this workshop
+  def potential_organizers
```

Comment:
This seems quite a lot of code for a controller method.  Could it still go into the model or into a helper class?

Reply:
done!